prdoc/pr_8473.prdoc

```diff
@@ -27,5 +29,7 @@ crates:
   validate: false
 - name: bridge-hub-westend-runtime
   bump: patch
+  validate: false
 - name: bridge-hub-westend-integration-tests
   bump: patch
+  validate: false
```

Comment:
nit:

```suggestion
- name: bridge-hub-westend-runtime
  bump: none
  validate: false
- name: bridge-hub-westend-integration-tests
  bump: none
  validate: false
```

test runtimes are not published so can be `bump: none` - as a general rule, if it has `westend` in the name, we don't publish.